must-gather/gather-olm-info

```diff
@@ -22,17 +22,25 @@ source "${SCRIPT_DIR}/common"
 
 BASE_COLLECTION_PATH=$1
 
-KEPLER_OPERATOR_NS="openshift-operators"
 KEPLER_OPERATOR_LABELS="operators.coreos.com/kepler-operator.openshift-operators"
+POWER_MONITORING_OPERATOR_LABELS="operators.coreos.com/power-monitoring-operator.openshift-operators"
```

Comment:
For the record I am okay with this but we need to address operator getting installed to `ns` other than `openshift-operators` (esp on 4.14+) in a different PR. Perhaps worth a bug @vprashar2929 ?

Reply:
if that is really needed, lets do it in another PR

Reply:
yes, as already mentioned :-)

Reply:
ok, can we merge this?